holidayextras/brands
# Adding in a bootstrap override for headings fonts, keeping out the li…

#### What does this PR do? (please provide any background)
There are some 'roboto' overrides happening as a result of applications which reference the toolkit. We need to add an override here and correctly referencing the font stack to avoid this cascading into applications.

#### What tests does this PR have?
None

#### How can this be tested?
Checkout this branch. `npm link ../brands` into the Pattern Library and you shouldn't see any trace of Roboto in the fonts.

#### Screenshots / Screencast
<img width="494" alt="screen shot 2016-03-30 at 10 33 33" src="https://cloud.githubusercontent.com/assets/6357978/14138043/df9386b6-f662-11e5-8bbf-f5e4073236ab.png">


#### What gif best describes how you feel about this work?
![](https://media.giphy.com/media/l2JHSLWYxzOdYF54s/giphy-tumblr.gif)

---


#### Reviewers

**Review 1**

**Review 2** \*

**Review 3** _(optional)_

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\*  for HX this review must be completed by an SE, SA or Project Guru
